core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/internal/Trace.java

```diff
@@ -1,48 +0,0 @@
-/*******************************************************************************
- * Copyright (c) 2003, 2011 IBM Corporation and others.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- *
- * Contributors:
- *     IBM Corporation - Initial API and implementation
- *******************************************************************************/
-package org.fusesource.ide.camel.model.service.core.internal;
-
-import org.eclipse.osgi.service.debug.DebugOptions;
-import org.eclipse.osgi.service.debug.DebugOptionsListener;
-import org.jboss.tools.foundation.core.plugin.AbstractTrace;
-
-/**
- * Helper class to route trace output.
- * The superclass keeps track of the debug options for your plugin, as well 
- * as when they change. The static methods in this class are here
- * only for convenience and easy of use
- */
-public class Trace extends AbstractTrace implements DebugOptionsListener {
```

Comment:
Why did u remove the Trace? To have that seems to be defacto standard in JBT afair.